### PR TITLE
Anti-Adblock Anti-RightClick: jootc.com

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/86220

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1246,6 +1246,7 @@ imslp.org#%#AG_onLoad(function(){var b=document.getElementById("sm_dl_wait");if(
 ||cdn.web.tv/asset/javascripts/smothScroll.min.js
 ||serialuk.net/js/copyright.js
 !--- CSS ---
+jootc.com#$#body, div, p, span, h1, h2, h3, h4, h5, h6, a, img { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; -khtml-user-select: auto!important; user-select: auto!important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1249]: As I said before, we don't use more than one selectors. These rules should be enough
```
jootc.com#$#html > body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; -khtml-user-select: auto!important; user-select: auto!important; }
jootc.com#$#html > body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; -khtml-user-select: auto!important; user-select: auto!important; }
```

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -449,7 +449,7 @@ pornissimo.org,pornobit.me,porntn.com,femdomtb.com,pornoham.me,tubsexer.com,home
 !
 ! ai_front anti-adblock
 ! #%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
-ubuntudde.com,mettablog.com,secondlifetranslations.com,charbelnemnom.com,lowkeytech.com,pinayvids.com,gcertificationcourse.com,game-owl.com,plugincrack.com,manyakan.com#%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
+jootc.com,ubuntudde.com,mettablog.com,secondlifetranslations.com,charbelnemnom.com,lowkeytech.com,pinayvids.com,gcertificationcourse.com,game-owl.com,plugincrack.com,manyakan.com#%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:452]: That's ok, but you forgot about content blockers. I will add it, please take a look.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: Okay. Thank you for your advice.
https://github.com/AdguardTeam/AdguardFilters/pull/86221/commits/339eb47f8524d7cc64ca9e06f08485eca64d034e

File: AnnoyancesFilter/sections/tweaks.txt [hunk 3]
```diff
@@ -1247,8 +1247,8 @@ imslp.org#%#AG_onLoad(function(){var b=document.getElementById("sm_dl_wait");if(
...
-page.kakao.com#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
-cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
+page.kakao.com,jootc.com#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
+cafe.daum.net,jootc.com#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1251]: They won't work in apps, I'll change it here.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**